lib/logger.js

```diff
@@ -227,6 +235,31 @@ Logger.prototype._bufferLog = function(message) {
     }
 };
 
+Logger.prototype._sendRequest = function(_config, dbgInfoAndAttemps, callback) {
+    axios(_config)
+        .then(function(response) {
+            if (response && response.status === 200) {
+                return {
```

Comment:
Here it doesn't return `callback` but in `L248`, it returns `callback` - so, the question is which one is correct - shouldn't this always return `callback`?

Reply:
oh good catch. Thank you!

Reply:
To your first comment: I see that CloudWatch retries to connect. I also retry to connect if it fails but I also save the failed lines in memory so they don't get lost.